Source/Core/DolphinQt/Debugger/BreakpointWidget.h

```diff
@@ -22,10 +22,10 @@ class BreakpointWidget : public QDockWidget
 
   void AddBP(u32 addr);
   void AddBP(u32 addr, bool temp, bool break_on_hit, bool log_on_hit, const QString& condition);
-  void AddAddressMBP(u32 addr, bool on_read = true, bool on_write = true, bool do_log = true,
-                     bool do_break = true);
-  void AddRangedMBP(u32 from, u32 to, bool do_read = true, bool do_write = true, bool do_log = true,
-                    bool do_break = true);
+  void AddAddressMBP(u32 addr, const QString& condition = {}, bool on_read = true,
+                     bool on_write = true, bool do_log = true, bool do_break = true);
+  void AddRangedMBP(u32 from, u32 to, const QString& condition, bool do_read = true,
+                    bool do_write = true, bool do_log = true, bool do_break = true);
```

Comment:
Small nitpick, feel free to ignore this comment: you might want to move the `condition` parameter at the end of the prototype for consistency with the `AddBP` prototype and it will also reduce the diff.